app/controllers/migration_controller.rb

```diff
@@ -7,19 +7,20 @@ def index; end
 
   def new
     @contribution = new_contribution
-    build_contribution_associations_unless_present(@contribution)
+    formify_contribution(@contribution)
   end
 
   def create
     @contribution = new_contribution
-    @contribution.assign_attributes(contribution_params)
+
+    assign_attributes_to_contribution(@contribution)
```

Comment:
No particular rationale behind that. We were however already doing so with the method renamed to `formify_contribution`.

Reply:
I don't have a particularly strong opinion about it, but if we're not  sending this `@contribution` object out of the controller, or are planning to call `assign_attributes_to_contribution` or `formify_contribution` with anything other than `@contribution`, then it seems unessecary to send the object around like this.
The way it is now, it might be easier to stub for testing though, if there were unit tests at method level for the two methods.

Reply:
I take your point but suggest we leave this for a future refactor.